css/.stylelintrc.json

```
@@ -0,0 +1,8 @@
+{
+	"extends": ["stylelint-config-airbnb", "stylelint-config-prettier"],
```

I think this needs added, I saw they were already in the package, unless airbnb is pulling them in. Might be good to be explicit anyway.
```suggestion
	"extends": ["stylelint-config-airbnb", "stylelint-config-prettier"],
	"plugins": ["stylelint-scss", "stylelint-order"],
```

---

Refactored into a new package in /plugins/stylelint-config-atlas. Check it out there. I've taken most of the rules specified here, but we are extending `stylelint-config-prettier` after we pull in `stylelint-config-atlas`. We need to do this to ensure rules don't (ever) conflict with prettier.

---

Configuration is here: https://github.com/microsoft/atlas-design/pull/6/files#diff-0d2baa706832c5370c7f6b8aa9f3e2fe8fe29c6f707f069fe9eea52058a0348d